File: heart_rate.py
def get_heart_rate(heart_rate_frame_data, fps):
    print('Begin HR')
    total_interval = heart_rate_frame_data[len(heart_rate_frame_data) - 1] - heart_rate_frame_data[0]
    avg_interval = total_interval / len(heart_rate_frame_data[1:])
    print('HR OK')
    return fps * 60 / avg_interval

File: heartRate.js
function getHeartRate(heartbeatFrameData, fps) {
    var totalInterval = heartbeatFrameData[heartbeatFrameData.length - 1] - heartbeatFrameData[0];
    var avgInterval = totalInterval / (heartbeatFrameData.length - 1);
    var heartRate = fps * 60 / avgInterval;
    return heartRate;
}

export default getHeartRate;
